### PR TITLE
Spookium and other Assorted Features

### PR DESCRIPTION
**Pull request by Shnupbups**
- Added Spookium Ore to the Spooky Dimension, smelts into Spookium Ingots
- Added Spookium Ingots, Nuggets and Blocks
- Added Reaper's Scythe craftable with Spookium, currently just a good weapon, will do fancy soul-reaping later
- Added Tainted Glass and Glass Panes, smelts from Tainted Sand
- Added Soul Glass and Glass Panes, smelts from Soul Sand
- Added Soul Bottles, made with Soul Glass, will be used for soul-reaping later
- Added loot tables for blocks that were missing them

### CODE REVIEW

**Review by Draylar — APPROVED** [review]
Looking good!

**Review by Shrimpstress — APPROVED** [review]
This gets an approval from me.

**Comment by Shrimpstress** [comment]
Keep in mind, stone in this dimension may change to be a custom type of stone, so the texture may need to be changed for Spookium Ore in the future.

File: src/main/java/com/fabriccommunity/spookytime/common/SpookyBlocks.java [hunk 1]
```diff
@@ -11,6 +13,7 @@
 import net.minecraft.item.Item;
 import net.minecraft.item.ItemGroup;
 import net.minecraft.sound.BlockSoundGroup;
+import net.minecraft.util.Rarity;
```
Comment (valoeghese) [inline src/main/java/com/fabriccommunity/spookytime/common/SpookyBlocks.java:16]: *cue 100 fabric community members arguing about one import, even though it's 100% necessary*

Comment (valoeghese) [reply src/main/java/com/fabriccommunity/spookytime/common/SpookyBlocks.java:16]: But tbh what does Rarity do

Comment (valoeghese) [reply src/main/java/com/fabriccommunity/spookytime/common/SpookyBlocks.java:16]: Does it mess with the coloured names

File: src/main/java/com/fabriccommunity/spookytime/common/SpookyWorldGen.java [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
 import com.fabriccommunity.spookytime.world.SpookyBiomeGroup;
 import com.fabriccommunity.spookytime.world.layer.AddSubBiomesLayer;
 
 import net.fabricmc.fabric.api.biomes.v1.OverworldBiomes;
 
+import net.minecraft.block.Blocks;
+import net.minecraft.util.registry.Registry;
 import net.minecraft.world.biome.Biome;
+import net.minecraft.world.gen.feature.DefaultFeatureConfig;
+import net.minecraft.world.gen.feature.Feature;
```
Comment (valoeghese) [inline src/main/java/com/fabriccommunity/spookytime/common/SpookyWorldGen.java:13]: These imports are unused?

File: src/main/java/com/fabriccommunity/spookytime/world/biome/SpookyForestBiome.java [hunk 3]
```diff
@@ -1,7 +1,10 @@
 package com.fabriccommunity.spookytime.world.biome;
 
+import com.fabriccommunity.spookytime.common.SpookyBlocks;
 import com.fabriccommunity.spookytime.common.SpookyEntities;
+import com.fabriccommunity.spookytime.common.SpookyWorldGen;
```
Comment (valoeghese) [inline src/main/java/com/fabriccommunity/spookytime/world/biome/SpookyForestBiome.java:5]: More unused imports?

**Review by valoeghese — COMMENTED** [review]
Unused imports?

File: src/main/java/com/fabriccommunity/spookytime/common/SpookyBlocks.java (hunk 1 above)
Comment (Shnupbups) [reply src/main/java/com/fabriccommunity/spookytime/common/SpookyBlocks.java:16]: Yes, rarities are how vanilla handles coloured names. See https://minecraft.gamepedia.com/Rarity

**Comment by Shnupbups** [comment]
Ah, yeah those are leftovers from my attempts to add carved pumpkins and Jack O lanterns to worldgen in the dimension, which I couldn't get to work.

**Review by valoeghese — APPROVED** [review]
Aside from unused imports this is fine :)

**Comment by Shrimpstress** [comment]
Other than the merge conflict of course.

**Comment by valoeghese** [comment]
Time for shnup to manually merge stuff :tiny_potato:

**Review by ImAK9 — APPROVED** [review]
Fix conflicts & it’s ready to ship!

**Pull request merged**